changelog.d/1.bugfix

```diff
@@ -0,0 +1 @@
+The parameters in the [test_events.py](synapse/tests/rest/client/v1/test_events.py) in the prepare method were in the wrong order. This change fixes the order. Contributed by Isaiah Singletary.
```

Comment:
Pleae rename this file to `7243.misc` (7243 to match the PR number and "misc" because it is a test only change.)

Reply:
Also, I think we can simplify it to say "Correct the parameters of a test fixture. Contributed by Isaiah Singletary."

Reply:
Looks good! Please delete this file and we can merge this.